charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+    {{- if .Values.ui.labels }}
+{{ toYaml .Values.ui.labels | indent 4}}
+    {{- end}}
+  name: {{ template "cp-schema-registry.fullname" . }}-ui
+spec:
+  replicas: {{ .Values.ui.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "cp-schema-registry.fullname" . }}
+      service: schema-registry-ui
+  template:
+    metadata:
+      {{- if .Values.ui.annotations }}
+      annotations: {{ toYaml .Values.ui.annotations | nindent 8 }}
+      {{- end }}
+      labels:
+        app: {{ template "cp-schema-registry.fullname" . }}
+        service: schema-registry-ui
+        {{- if .Values.ui.labels }}
+{{ toYaml .Values.ui.labels | indent 8}}
+        {{- end}}
+    spec:
+      {{- if .Values.securityContext }}
+      securityContext: {{ toYaml .Values.securityContext | nindent 8 }}
+      {{- end}}
+      containers:
+        - env:
+            - name: PROXY
+              value: "true"
+            - name: SCHEMAREGISTRY_URL
+              value: {{ template  "cp-schema-registry.connectUrlValue" . }}
+          image: landoop/schema-registry-ui
```

Comment:
It is a safe registry?

Reply:
Depends what you mean by safe :) 
It's a registry that has been around for many years, backed by this company https://lenses.io/
But I can build an image from it and store it in our own bbc registry if you think it's relevant.